### DISCUSSION TITLE
Duvidas sobre react

### DISCUSSION THREAD

**Discussion by Diog010101** [Diog010101/Diog010101#1] [Ideas]
Qual a diferença entre react e rect native?

**Comment by giovanni060912** [comment]
Não a tantas diferenças entre as stacks o que muda é o fato de uma ser utilizada na web e a outra em projetos mobile

**Comment by Diog010101** [comment]
Enquanto ao vue js oq pode se dizer?

**Reply by giovanni060912** [reply]
Nesse caso ainda é um framework javascript com regras diferentes do react com um ciclo diferente com algumas semelhanças

**Reply by Diog010101** [reply]
> Nesse caso ainda é um framework javascript com regras diferentes do react com um ciclo diferente com algumas semelhanças

certo mais tem muitas diferenças?

**Comment by giovanni060912** [comment]
Nesse caso muda algumas coisas como o hooks que existem no react e no react native ja não existen no vue entre outros detalhes

**Comment by biaBarreto** [comment]
Eu sou nova na area e tbm tinha duvidas como essa que agr estão bem mais claras
